tests/test_filters.py

```diff
@@ -963,7 +963,7 @@ class _CustomFilter(BaseFilter):
             _CustomFilter()
 
     def test_custom_unnamed_filter(self, update):
-        class Unnamed(BaseFilter):
+        class Unnamed(MessageFilter):
```

Comment:
We should probably test that UpdateFilter works as desired too?

Reply:
Testing with both via fixture, as well in `conftest.py`. Don't know if my fixture is the best … but it works …

Reply:
Yea this seems pretty good for now.
We should probably be testing each individual filter, but this is good enough for now :)